e2e/pnpm_workspace/snapshots/defs.bzl

```diff
@@ -104,19 +108,8 @@ def npm_link_all_packages(name = "node_modules", imported_links = [], prod = Tru
             tags = ["manual"],
         )
         _npm_local_package_store(
-            package_store_name = "@lib+b_alias@0.0.0",
```

Comment:
This was an alternate name linking to `@lib/b` which caused a duplicate `local_package_store` target. Now the `_fp_link_3("@lib/b_alias")` below simply links to the original `@lib/b` without the duplication 👍